Rsa netwitness mirroring (#29220)
demisto/content: Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_0.md

#### Integrations

##### RSANetWitness v11.5

> - Added the incident mirroring functionality with the **Incident Mirroring Direction** and **Mirror closure** parameters.

Reviewer: ```suggestion
Added the incident mirroring functionality with the **Incident Mirroring Direction** and **Mirror closure** parameters.
```

#### Layouts

##### RSA NetWitness Layout

> - Rename the `Alerts` tab into `SIEM` with multiple section to better manage and investigate the incident in XSOAR.

Reviewer: ```suggestion
Renamed the `Alerts` tab to `SIEM` with multiple sections to better manage and investigate the incident in Cortex XSOAR.
```

#### Mappers

##### RSA NetWitness v11.5 - incoming mapper

> - Added mirroring mandatory fields: **mirror_instance**, **mirror_direction** and **incident_id**

Reviewer: ```suggestion
Added the following mirroring mandatory fields:
- **mirror_instance**
- **mirror_direction**
- **incident_id**
```

#### Scripts

##### SetRSANetWitnessAlertsMD

> - Remove header of the markdown for better UI consistency

Reviewer: ```suggestion
Remove the header of the markdown for better UI consistency.
```

##### RSAGetRawLog

> - Get the raw log and metas of events using RSA Netwintess Packets & Logs

Reviewer: ```suggestion
Gets the raw log and metas of events using RSA NetWitness Packets & Logs.
```

##### RSADisplayMetasEvents

> - Display alert event metas at the origin of the incident

Reviewer: ```suggestion
Displays alert event metas at the origin of the incident.
```

#### Integrations

##### RSANetWitness v11.5

> Added the incident mirroring functionality with the **Incident Mirroring Direction** and **Mirror closure** parameters.

Reviewer 2: ```suggestion
- Added the incident mirroring functionality with the **Incident Mirroring Direction** and **Mirror closure** parameters.
- Updated the Docker image to: *demisto/python3:3.10.13.73190*.
```

#### Layouts

##### RSA NetWitness Layout

Renamed the `Alerts` tab to `SIEM` with multiple sections to better manage and investigate the incident in Cortex XSOAR.

#### Mappers

##### RSA NetWitness v11.5 - incoming mapper

Added the following mirroring mandatory fields:
- **mirror_instance**
- **mirror_direction**
- **incident_id**

#### Scripts

##### SetRSANetWitnessAlertsMD

> Remove the header of the markdown for better UI consistency.

Reviewer 2: ```suggestion
- Remove the header of the markdown for better UI consistency.
- Updated the Docker image to: *demisto/python3:3.10.13.73190*.
```

#### Integrations
##### RSANetWitness v11.5

- Added the incident mirroring functionality with the **Incident Mirroring Direction** and **Mirror closure** parameters.
- Updated the Docker image to: *demisto/python3:3.10.13.73190*.

<!--
##### RSA NetWitness v11.1 (Deprecated)
- Remove non standard ' in the yml file

> -->

Author: if I remove it, some test will failed, are you sure ?

> -->

Author: it's done
